scripts/populate-instance.sh

```diff
@@ -62,14 +62,22 @@ apache_wsgi_restart () {
     $sudo -u "${INVENIO_WEB_USER}" touch "${INVENIO_WEB_DSTDIR}/var/www-wsgi/invenio.wsgi"
 }
 
-start_apache_ubuntu_precise () {
+start_apache_ubuntu12 () {
     $sudo /usr/sbin/service apache2 start
 }
 
-stop_apache_ubuntu_precise () {
+stop_apache_ubuntu12 () {
     $sudo /usr/sbin/service apache2 stop
 }
 
+start_apache_ubuntu14 () {
```

Comment:
If both are the same, why not `[start|stop]_apache_ubuntu`, IMHO it might make readability of the script a bit easier